.github/workflows/publish.yml

```diff
@@ -6,26 +6,42 @@ jobs:
     strategy:
       fail-fast: false
       matrix: # Windows-2022 has been temporarily removed. We are unable to build 2x windows installs at the same name due to naming convention issues. Issue made upstream here: https://github.com/tauri-apps/tauri-action/issues/215
-        platform: [macos-latest, ubuntu-latest, windows-2019]
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-2019
+          - windows-2022 # test again to see if it will fail, comment out this again if necessary
 
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: actions/checkout@v2
+      - name: git checkout
+        uses: actions/checkout@v2
+
       - name: setup node
         uses: actions/setup-node@v1
         with:
           node-version: 14
-      - name: install Rust stable
+
+      - name: install Rust nightly
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly-2021-12-14
+          override: true
+          components: rustfmt, clippy
+
+        # Workaround to resolve link error with C:\msys64\mingw64\bin\libclang.dll
+      - name: Remove msys64
+        run: Remove-Item -LiteralPath "C:\msys64\" -Force -Recurse
+        if: runner.os == 'Windows'
+
       - name: install webkit2gtk (ubuntu only)
         if: matrix.platform == 'ubuntu-latest'
         run: |
           sudo apt-get update
           sudo apt-get install -y --no-install-recommends libwebkit2gtk-4.0-dev libappindicator3-dev0
 
       # Imports PFX Certificate into keystore, allows Tauri to then sign the exe without being passed the private key.
+      # This normally should not be required on CI-Tests, but due to the way that tauri has its code signing configuration for windows it is required that we sign or the build will fail.
```

Comment:
This message is incorrectly copy-pasted from another place, but it is confusing here because this is a publishing action, it has to import certificate to sign release later.